virt/virt-4-13-release-notes.adoc

```diff
@@ -84,6 +84,7 @@ The SVVP Certification applies to:
 === Web console
 
 //CNV-23474 Release note: NEW Paste clipboard into VNC
+You can now xref:../virt/virt-web-console-overview.adoc#ui-virtualmachine-details-console_virt-web-console-overview[paste a string] from your client’s clipboard into the guest when using the VNC console.
```

Comment:
Should this be a bullet like its peers in other sections?
```suggestion
* You can now xref:../virt/virt-web-console-overview.adoc#ui-virtualmachine-details-console_virt-web-console-overview[paste a string] from your client’s clipboard into the guest when using the VNC console.
```

Reply:
Whoops, yup!